packages/ncids-css/patches/@uswds+uswds+3.3.0.patch

```diff
@@ -53,7 +53,7 @@ index cda8dbc..d0db4b2 100644
 +++ b/node_modules/@uswds/uswds/packages/uswds-core/src/styles/settings/_settings-color.scss
```

Comment:
can you comment on the lines you changed in this PR since this is happening again